### DISCUSSION TITLE
update from vaxilu/x-ui to FranzKafkaYu/x-ui

### DISCUSSION THREAD

**Discussion by void1995** [FranzKafkaYu/x-ui#149] [Q&A] [answered]
hi , i have /vaxilu/x-ui  already installed , can i update my panel  to yours with FranzKafkaYu/x-ui/master/install.sh script or i have to reinstall it ?

**Comment by FranzKafkaYu** [comment, marked as answer]
> hi , i have /vaxilu/x-ui already installed , can i update my panel to yours with FranzKafkaYu/x-ui/master/install.sh script or i have to reinstall it ?

@void1995 of course you can update by this script directly.And install.sh is used for Chinese version,if you need English version, plz use install_en.sh script.

**Reply by void1995** [reply]
what about 3 limit maximum number of users per port , can we increase this amount per port?

**Comment by Recoba86** [comment]
Hi
when I update to your version
all info like user and their info from vaxilu panel will come to Franzkafka panel?

**Reply by Recoba86** [reply]
and do I need to just run this code ?!
`bash <(curl -Ls https://raw.githubusercontent.com/FranzKafkaYu/x-ui/master/install_en.sh)`

**Reply by FranzKafkaYu** [reply]
> and do I need to just run this code ?! `bash <(curl -Ls https://raw.githubusercontent.com/FranzKafkaYu/x-ui/master/install_en.sh)`

yes,and all data info will be keeped.
